fee_allocator/accounting/decorators.py

```diff
@@ -0,0 +1,56 @@
+from functools import wraps
+from decimal import Decimal
+
+
+def return_zero_if_dust(threshold=Decimal("1E-20"), any_or_all="any"):
```

Comment:
Why do we want to do this?  Is this about rounding down, we should be very very considerate of rounding logic, it should always be quite apparent what is happening.  I don't think I see this being used anywhere, can it be removed?  Would prefer rounding was more explicit where it happened.

Reply:
originally i decorated most of the PoolFee and CorePoolChain attributes/properties that returned decimals to use this to avoid having a bunch of repetitive code on each method, but ultimately chose to do the rounding at the end in the FeeAllocator. it is not used anywhere atm, but i didnt see any harm in deleting it